src/capture-view.ts

```diff
@@ -77,8 +77,12 @@ export default class CaptureView extends View {
   }
 
   async getDevices() {
-    let devices = await navigator.mediaDevices.enumerateDevices() as MediaDeviceInfo[];
-    devices = devices.filter((device) => device.kind === 'videoinput');
+    let devices: MediaDeviceInfo[] = [];
+
+    if ('mediaDevices' in navigator) {
```

Comment:
Instead of having `'mediaDevices' in navigator` checks everywhere in the codebase, it would be nice to have a constants file like https://github.com/GoogleChrome/sample-media-pwa/blob/master/src/client/scripts/constants/constants.js where we would have a `SUPPORTS_MEDIA_DEVICES` constant for instance?